File: shared/src/types/sound.ts
export type SoundType =
  | 'card_deal'
  | 'card_flip'
  | 'chip_bet'
  | 'chip_win'
  | 'check'
  | 'fold'
  | 'all_in'
  | 'timer_warning'
  | 'your_turn';

export type SoundCategory = 'cards' | 'chips' | 'actions' | 'notifications';

export const ALL_SOUND_CATEGORIES: SoundCategory[] = ['cards', 'chips', 'actions', 'notifications'];

export const SOUND_CATEGORIES: Record<SoundType, SoundCategory> = {
  card_deal: 'cards',
  card_flip: 'cards',
  chip_bet: 'chips',
  chip_win: 'chips',
  check: 'actions',
  fold: 'actions',
  all_in: 'actions',
  timer_warning: 'notifications',
  your_turn: 'notifications',
};

File: ios/PokerSofta/Sources/Models/Sound.swift
import Foundation

enum SoundType: String, Codable {
    case cardDeal = "card_deal"
    case cardFlip = "card_flip"
    case chipBet = "chip_bet"
    case chipWin = "chip_win"
    case check, fold
    case allIn = "all_in"
    case timerWarning = "timer_warning"
    case yourTurn = "your_turn"
}
